buildings/exocomp-factory.json

```diff
@@ -0,0 +1,1824 @@
+      ],
+      "rewards": []
+    },
+    {
+      "bonuses": {
+        "bonus1": 19.0,
+        "bonus2": 2.0,
+        "bonus3": 2.0
+      },
+      "build_costs": {
+        "materials": [],
+        "others": [
+          {
+            "type": "Axionic Servo",
+            "value": 17
+          }
+        ],
+        "resources": [
+          {
+            "type": "tritanium",
+            "value": 275
+          },
+          {
+            "type": "parsteel",
+            "value": 9350
+          }
+        ]
+      },
+      "build_time": 18300,
+      "increased_power": 525,
+      "level": 12,
+      "requirements": [
+        {
+          "name": "Operations",
+          "level": 16
+        }
+      ],
+      "rewards": []
+    },
+    {
+      "bonuses": {
+        "bonus1": 19.0,
+        "bonus2": 2.0,
+        "bonus3": 2.0
+      },
+      "build_costs": {
+        "materials": [],
+        "others": [
+          {
+            "type": "Axionic Servo",
+            "value": 19
+          }
+        ],
+        "resources": [
+          {
+            "type": "tritanium",
+            "value": 400
+          },
+          {
+            "type": "parsteel",
+            "value": 13600
+          }
+        ]
+      },
+      "build_time": 27360,
+      "increased_power": 650,
+      "level": 13,
+      "requirements": [
+        {
+          "name": "Operations",
+          "level": 16
+        }
+      ],
+      "rewards": []
+    },
+    {
+      "bonuses": {
+        "bonus1": 22.0,
+        "bonus2": 2.0,
+        "bonus3": 2.0
+      },
+      "build_costs": {
+        "materials": [],
+        "others": [
+          {
+            "type": "Axionic Servo",
+            "value": 22
+          }
+        ],
+        "resources": [
+          {
+            "type": "tritanium",
+            "value": 550
+          },
+          {
+            "type": "parsteel",
+            "value": 18700
+          }
+        ]
+      },
+      "build_time": 38160,
+      "increased_power": 800,
+      "level": 14,
+      "requirements": [
+        {
+          "name": "Operations",
+          "level": 16
+        }
+      ],
+      "rewards": []
+    },
+    {
+      "bonuses": {
+        "bonus1": 22.0,
+        "bonus2": 2.0,
+        "bonus3": 3.0
+      },
+      "build_costs": {
+        "materials": [],
+        "others": [
+          {
+            "type": "Axionic Servo",
+            "value": 43
+          }
+        ],
+        "resources": [
+          {
+            "type": "tritanium",
+            "value": 750
+          },
+          {
+            "type": "parsteel",
+            "value": 25500
+          }
+        ]
+      },
+      "build_time": 48960,
+      "increased_power": 975,
+      "level": 15,
+      "requirements": [
+        {
+          "name": "Operations",
+          "level": 16
+        }
+      ],
+      "rewards": [
+        {
+          "type": 3906625240,
+          "value": 1
+        }
+      ]
```

Comment:
Also parsing problem for rewards in level 15.

Reply:
Yeah I'm resolving dependencies that apparently exist in rewards 🙈